app/controllers/testOnly/UserResearchLoginController.scala

```diff
@@ -56,6 +56,7 @@ class UserResearchLoginController @Inject()(mcc: MessagesControllerComponents,
   private[testOnly] def buildGGSession(authExchange: AuthLoginAPIResponse): Session = Session(Map(
     SessionKeys.sessionId -> (if (authExchange.sessionId.isEmpty) SessionId(s"session-${UUID.randomUUID}").value else authExchange.sessionId),
     SessionKeys.authToken -> authExchange.token,
-    SessionKeys.lastRequestTimestamp -> DateTime.now.getMillis.toString
+//    SessionKeys.lastRequestTimestamp -> DateTime.now.getMillis.toString
```

Comment:
can we please remove this commented line

Reply:
Will remove in the next commit